solidstart-tanstackquery-tailwind-modules/src/components/UserProfile/data.ts

```diff
@@ -0,0 +1,20 @@
+export const userProfileProps = {
```

Comment:
Maybe it could be better to call this `userProfileMockedData`? We have a type called `UserProfileProps` and could be confusing.